content/en/docs/ambient/getting-started/setup/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Install Ambient
+description: Download and Install Istio in Ambient mode.
+weight: 1
+owner: istio/wg-networking-maintainers
+test: yes
+---
+
+In this section you'll download Istio CLI (`istioctl`) and install Istio in ambient mode on your Kubernetes cluster. You'll need a Kubernetes cluster to proceed. If you don't have a cluster, you can use [kind](/docs/setup/platform-setup/kind) or any other [supported Kubernetes platform](/docs/setup/platform-setup).
+
+## 1. Download Istio CLI
+
+You'll use [Homebrew](https://brew.sh/) to download the Istio CLI:
```

Comment:
```suggestion
You'll use [Homebrew](https://brew.sh/) to download the Istio CLI:
```

Personally don't like doing this, we should be encouraging users to download the binary from us with a version that matches what their cluster is running. We should link to this as a pre-req, or mirror the same instructions for consistency. https://istio.io/latest/docs/setup/getting-started/#download

Reply:
One of the things I tried to achieve with these instructions is to simplify them and get the reader ready to go as fast as possible. Executing a download script that downloads a compressed file, unpacks it into the current folder then asks you to set up your dotfiles/path to make it useful doesn't help with simplicity. The fact that we're not using either Brew or have a simpler installation might be a separate thing that could be improved. 

Note that brew automatically and on a schedule checks for outdated packages and updates them. (e.g. the 1.22 was out on May 13th and the brew formula was updated on the same day as well) 

If the consensus is that we should revert to the current instructions, I can do that, but I think we could do much better with simplifying the installation experience.

Reply:
I support the goal of making this easier, but I also need to point out that using we can't assume people are using a Mac.  For now, until we decide a larger change, let's use the standard "download CLI" instructions.

Reply:
```(e.g. the 1.22 was out on May 13th and the brew formula was updated on the same day as well)```

This illustrates my point. istioctl being upgraded ahead of the cluster can introduce some unexpected or bad behavior. Given the upgrade trends we see when Mitch did his surveys, many users are at least N-1, some further. istioctl should not be on a different minor version.

> I support the goal of making this easier, but I also need to point out that using we can't assume people are using a Mac. For now, until we decide a larger change, let's use the standard "download CLI" instructions.



> I support the goal of making this easier, but I also need to point out that using we can't assume people are using a Mac. For now, until we decide a larger change, let's use the standard "download CLI" instructions.

Homebrew is not just for Macs. I use it on my Linux systems. My point is consistency across docs, and that the version of istioctl should match the version of Istio installed in the clusters, something `brew` cannot determine on its own.

Reply:
Are we expecting upgrading users to go through the "Getting Started" page? Perhaps we need a section that's a quick start guide or something aimed at new users?

My assumption was that someone going through the "getting started" page probably doesn't have Istio installed at all. But I might be wrong.

Reply:
What about after getting started? IE, if brew upgrades istioctl, but Istio in their env is not upgraded. Manifests are baked into istioctl, and they're specific to that minor version. Setting someone up for a bad time, imo.

Reply:
Brew won't automatically update it, user has to run update/upgrade, but I see your point. 

I am not necessarily advocating for Brew per-se, but for an easier way to install the CLI. E.g. even if it was an existing downloadScript that downloads the binary only and puts it in ~/.istio folder or something and doesn't require me to set the path and it doesn't "pollute" my current folder with yaml manifests.

Reply:
Resolving this - we're keeping the existing installation method in the guide. If there are changes that we want to make, they should be made in the installer first.